docs/source/features/heterogeneous-gpu.rst

```diff
@@ -20,6 +20,15 @@ There are three main components in Heterogeneous GPU Inference Feature: (1) LLM
 Example
 -------
 
+Preparation: Enable related components, including request tracking at the gateway.
```

Comment:
Seems the change is straightforward,  can we add an alternative here. for example, ask user to 

```
kubectl edit deployment aibrix-gateway-plugin

then append env here.
            # - name: AIBRIX_GPU_OPTIMIZER_TRACING_FLAG
            #   value: "true"
```

user can also use  built-in manifest ` kubectl apply -k config/experimentals/gpu-optimizer`